xonsh/history/__init__.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+"""Implements the xonsh history object."""
+import builtins
+import time
+
+from xonsh.history.dummy import History as DummyHistory
+from xonsh.history.json import History as JsonHistory
+from xonsh.history.json import history_main
+from xonsh.history.sqlite import History as SqliteHistory
```

Comment:
Also, when you amalgamate, these class names should be changed to their aliases so that they don't override each other.